conf/awsbatch.config

```diff
@@ -0,0 +1,14 @@
+//Nextflow config file for running on AWS batch
+
+params {
+  config_profile_name = 'AWSBATCH'
+  config_profile_description = 'AWSBATCH Cloud Profile'
+  config_profile_contact = 'Alexander Peltzer (@apeltzer)'
+  config_profile_url = 'https://aws.amazon.com/batch/'
+}
+
+aws.region = params.awsregion
```

Comment:
This line causes the tests to fail 😱 

https://travis-ci.com/nf-core/configs/jobs/258237834#L362